api/src/utils/pricing.ts

```diff
@@ -1,13 +1,12 @@
 import { averageHoursInAMonth } from "./constants";
 
-
 export function getAkashPricing(cpu: number, memory: number, storage: number) {
   // Pricing obtained from https://github.com/akash-network/helm-charts/blob/main/charts/akash-provider/scripts/price_script_generic.sh
   const targetMemory = 1.25; // USD/GB-month
   const targetHDEphemeral = 0.08; // USD/GB-month
-  const targetHDPersHDD = 0.1; // USD/GB-month
-  const targetHDPersSSD = 0.12; // USD/GB-month
-  const targetHDPersNVME = 0.14; // USD/GB-month
+  // const targetHDPersHDD = 0.1; // USD/GB-month
```

Comment:
It wasn't used?

Reply:
We only compare ephemeral storage pricing, no persistent storage for now